doc/CONTRIBUTING.md

```diff
@@ -20,6 +20,12 @@ Here are the basic steps to submit a pull request.
 
 1. Make the test pass.
 
+1. Run linters and fix any linting errors they brings up.
+   1. `bundle exec standardrb --fix` is required by CI
+   1. But you should also be a good citizen and run:
```

Comment:
I would leave out the "good citizen" part, we have contributors who are not (USA) citizens :)

Reply:
Hey, they can still be good internet citizens! Definitely not intending to relate it to US citizenship :)